ledger/internal/appcow.go

```diff
@@ -443,7 +443,7 @@ func (cb *roundCowState) DelKey(addr basics.Address, aidx basics.AppIndex, globa
 
 // MakeDebugBalances creates a ledger suitable for dryrun and debugger
 func MakeDebugBalances(l LedgerForCowBase, round basics.Round, proto protocol.ConsensusVersion, prevTimestamp int64) apply.Balances {
-	base := makeRoundCowBase(l, round-1, 0, basics.Round(0), config.Consensus[proto])
+	base := makeRoundCowBase(l, round, 0, basics.Round(0), config.Consensus[proto])
```

Comment:
@algorandskiy I had to make this change for the tealdbg tests to work. Does this change seem okay or will it break stuff?

Reply:
I need to look, but the idea is when block X is evaluated, the blockchain has X-1 persisted so cow base gets created for X-1. I guess tealdbg tests need adjustment.

Reply:
@algorand/scytale any thoughts?